deployment/docker/README.md

```diff
@@ -0,0 +1,39 @@
+# OODT-docker
+
+Automatic RADIX build for Apache OODT
+
+## Usage
+
+You should edit docker file replacing the parameters below with whatever you want before build the image.
+
+1. The groupId is a place to specify your company's namespace. 
+    ```dockerfile
+    -DgroupId = com.mycompany
+    ```
+2. The artifactId is a place to specify a short name of your project. 
+    ```dockerfile
+    -DartifactId = oodt
+    ```
+3. The version indicates the initial version label for your project. 
+    ```dockerfile
+    -Dversion = 0.1
+    ``` 
+4. The oodt flag indicates the version of OODT that you want your project to be built on. N.B., this should most likely match [the most recent version of OODT](https://search.maven.org/search?q=g:org.apache.oodt).
+    ```dockerfile
+    -Doodt = 1.2.5
+    ```
+
+### Building OODT Docker Image
+Execute below command in a prompt which is opened where the Docker file placed. Replace any thing with `oodt_docker` as it is the name of your docker image.
+```dockerfile
+sudo docker build -t oodt_docker . 
+```    
+
+### Running OODT Docker Image
+After your docker image successfully built, execute below prompt command.`oodt_docker` is the oodt docker image name.
+```dockerfile
+sudo docker run -p 8080:8080 -name my_first_oodt -i -t oodt_docker
```

Comment:
Did you mean expose other services separately? 
as an example `sudo docker run -p 9000:9000 --name my_first_oodt -i -t oodt_docker` for start FileManager service.

Reply:
@NGimhana No. Exposing them in a single command. Like
`sudo docker run -p 8080:8080 -p 9000:9000 -p 9001:9001 -name my_first_oodt -i -t oodt_docker` and etc. This will be helpful if a user wants to inject files to file manager and query just to play around. Later we can add capability to mount volumes for directories where data are stored (may be under the GSoC project).

Reply:
@IMS94, fixed as requested. Would you please review it.